packages/node/test/integration/utils.ts

```diff
@@ -450,12 +450,21 @@ export async function collateralizeChannel(
 
 export async function createChannel(nodeA: Node, nodeB: Node) {
   return new Promise(async (resolve, reject) => {
+    let nodeBChannelCreationAck = false;
     nodeA.on(NODE_EVENTS.CREATE_CHANNEL, async () => {
       expect(await getInstalledAppInstances(nodeA)).toEqual([]);
       expect(await getInstalledAppInstances(nodeB)).toEqual([]);
+
+      while (!nodeBChannelCreationAck) {
+        await sleep(500);
```

Comment:
This is a wacky loop. Why do this? Just only resolve the promise after the event returns in side this block:

```ts
 nodeB.once(NODE_EVENTS.CREATE_CHANNEL, async () => {
      nodeBChannelCreationAck = true;
// resolve() here?
    });
```